solarforecastarbiter/datamodel.py

```diff
@@ -870,7 +870,7 @@ class ForecastObservation(BaseModel):
     __post_init__ checking.
     """
     forecast: Forecast
-    observation: Observation
+    observation: Union[Observation, Aggregate]
```

Comment:
observation can be observation or aggregate? what about a ``ForecastAggregate`` object and then report can take either of those. Then processing the ``object_pairs``like you suggested probably works better

Reply:
Was just wondering about that too. Could we keep the attribute  `observation` or would you want to change it to `aggregate`? Lots of code reaches for that attribute.

Reply:
probably change it to ``aggregate`` for clarity. Maybe the base class of ``ForecastObservation`` and ``ForecastAggregate`` has a property like ``data_object`` that is either an observation or aggregate for those functions that don't care about the difference.